typings/vscode-mssql.d.ts

```diff
@@ -569,10 +575,16 @@ declare module 'vscode-mssql' {
         connectionStringSetting: string;
     }
 
+    /**
+     * Parameters for getting the names of the Azure functions in a file
+     */
     export interface GetAzureFunctionsParams {
         filePath: string;
     }
 
+    /**
+     * Result from a get Azure functions request
```

Comment:
May want to consider adding some comments for the parameters as well. For example on 582 for filePath what types of "paths" does it support? Relative/absolute? What about file share?

And for this one - what are the string values? (this goes back to my earlier comment about either renaming this to azureFunctionNames or having it return an object that has a name property). But at least adding a comment explaining that these are the names of the functions is an ok start.